test/ExpressionSet_basic_test.py

```diff
@@ -132,30 +39,32 @@ def test_save_expression_set(
 
 
 def test_save_expression_set_mismatched_genomes(
-    test_data: dict,
     ws_id: int,
     clients: dict[str, Any],
     set_api_client: SetAPI,
     context: dict[str, str | list],
+    dummy_shock_file_handle: str,
+    genome_refs,
+    annotation_ref,
+    alignment_refs,
 ) -> None:
     expression_set_name = "expression_set_bad_genomes"
     expression_set = {
         "description": "this_better_fail",
         "items": [
             {
                 "ref": make_fake_expression(
-                    os.environ["SDK_CALLBACK_URL"],
-                    test_data["dummy_path"],
+                    dummy_shock_file_handle,
```

Comment:
instead of doing this shock transaction every time, just create a fixture for the dummy shock file handle and use that